pkg/ai/bedrock_support/completions.go

```diff
@@ -50,7 +64,27 @@ type AmazonCompletion struct {
 	completion ICompletion
 }
 
+func isModelSupported(modelName string) bool {
+	for _, supportedModel := range SUPPPORTED_BEDROCK_MODELS {
+		if modelName == supportedModel {
+			return true
+		}
+	}
+	return false
+}
+
 func (a *AmazonCompletion) GetCompletion(ctx context.Context, prompt string, modelConfig BedrockModelConfig) ([]byte, error) {
```

Comment:
I see where you're going with this, creating a generic Amazon completion, which I am not against. It's something I'll have to remember to work with!